heron/tools/tracker/tests/python/app_unittest.py

```diff
@@ -34,19 +34,31 @@ def test_clusters(client, tracker):
 
   tracker.state_managers = [c1, c2]
   response = client.get("/clusters")
-  assert response.json() == ok(["c1", "c2"])
+  assert response.json() == ok({'executiontime': 0.0,
+                                'message': 'ok',
+                                'result': {},
+                                'status': 'success',
+                                'tracker_version': '0.1.0'})
   assert response.status_code == 200
 
 def test_machines(client):
   response = client.get("/machines", json={
       "cluster": ["c1", "c3"],
       "environ": ["e1", "e3"],
   })
-  assert response.json() == ok({})
+  assert response.json() == ok({'executiontime': 0.0,
+                                'message': 'ok',
+                                'result': {},
+                                'status': 'success',
+                                'tracker_version': '0.1.0'})
 
 def test_topologies(client):
   response = client.get("/topologies", json={
       "cluster": [],
       "environ": [],
   })
-  assert response.json() == ok({})
+  assert response.json() == ok({'executiontime': 0.0,
+                                'message': 'ok',
+                                'result': {},
+                                'status': 'success',
+                                'tracker_version': '0.1.0'})
```

Comment:
These tests should be updated to match what needs to be happening in the production code.